requirements/edx/github.txt

```diff
@@ -47,7 +47,7 @@
 
 # Third-party:
 git+https://github.com/cyberdelia/django-pipeline.git@1.5.3#egg=django-pipeline==1.5.3
-git+https://github.com/edx/django-wiki.git@v0.0.9#egg=django-wiki==0.0.9
+git+https://github.com/edx/django-wiki.git@eb5a02c1cf7847f091ed3fc8159ea6f034102b2c#egg=django-wiki==0.0.10
```

Comment:
After https://github.com/edx/django-wiki/pull/23 is merged, I will update this to just reference 0.0.10.